_topic_map.yml

```diff
@@ -2190,6 +2190,8 @@ Topics:
     File: virt-create-vms
   - Name: Editing virtual machines
     File: virt-edit-vms
+  - Name: Scheduling virtual machines
```

Comment:
This is a very advanced feature - I'd move it to an advanced chapter.

Reply:
Scheduling virtual machines is now available in the advanced virtual machine management section.